app/assets/stylesheets/components.scss

```diff
@@ -251,9 +251,11 @@
 }
 
 .primary-checkpoint-container {
+  bottom: 1rem;
   cursor: pointer;
-  display: block;
+  display: inline-block;
   font-size: 22px;
+  left: 7rem;
```

Comment:
Why was this required? If you require it to align the checkbox better; please add a separate class to this particular checkbox and move the style there. It can be saved in app/assets/stylesheets/users.scss
Changing this on main checkbox class would break other ones on platform.

Reply:
because i moved h1 above label somehow checkbox lost its alignment so to make it align done this